alerts.js

```diff
@@ -0,0 +1,14 @@
+
+// ruleid:javascript-prompt
+var name = prompt('what is your name');
+// ruleid: javascript-alert
+alert('your name is ' + name);
+alert('not', 'a', 'valid', 'alert')
+// ruleid: javascript-confirm
+if ( confirm("pushem!") == true) {
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Found conform() call'.
 
found confirm() call; should this be in production code?
 
Severity: Medium
 <!-- 1f157063-1adf-470f-8053-56bbd285d1c4 -->